x-pack/plugins/fleet/common/constants/authz.ts

```diff
@@ -5,22 +5,123 @@
  * 2.0.
  */
 
-export const ENDPOINT_PRIVILEGES = [
-  'writeEndpointList',
-  'readEndpointList',
-  'writeTrustedApplications',
-  'readTrustedApplications',
-  'writeHostIsolationExceptions',
-  'readHostIsolationExceptions',
-  'writeBlocklist',
-  'readBlocklist',
-  'writeEventFilters',
-  'readEventFilters',
-  'writePolicyManagement',
-  'readPolicyManagement',
-  'writeActionsLogManagement',
-  'readActionsLogManagement',
-  'writeHostIsolation',
-  'writeProcessOperations',
-  'writeFileOperations',
-] as const;
+import { DEFAULT_APP_CATEGORIES } from '@kbn/core-application-common';
+
+const SECURITY_SOLUTION_APP_ID = 'siem';
+
+export const ENDPOINT_PRIVILEGES = {
```

Comment:
added some comments but basically needed to add some mappings since `readSecuritySolution` and `writeSecuritySolution` aren't fetched from kibana quite the same as the other privileges: https://github.com/joeypoon/kibana/blob/4c8dd903f6ae89941bb60dda5428af6da9a27b7e/x-pack/plugins/fleet/server/services/security/security.ts#L71